library/src/main/java/com/pokegoapi/main/CommonRequest.java

```diff
@@ -104,4 +108,50 @@ public static GetInventoryMessage getDefaultGetInventoryMessage(PokemonGo api) {
 				CommonRequest.getDownloadSettingsMessageRequest(api));
 		return serverRequests;
 	}
+
+
+	/**
+	 * List of common requests
+	 *
+	 * @param api The current instance of PokemonGO
+	 * @return an array of ServerRequest
+	 */
+	public static ServerRequest[] commonRequests(PokemonGo api) {
+		ServerRequest[] serverRequests = new ServerRequest[4];
```

Comment:
i prefer to leave it as it is if you don't mind.
In the future can be possible that common requests will be  much more than 4.
i was thinking of adding something like
`
final int GET_HATCHED_EGGS_CR = 0;
final int GET_INVENTORY_CR = 1;
final int CHECK_AWARDED_BADGES_CR = 2;
`
and so on... 
this will make the parse method easy to read and implement with something like

```
 switch()
 case GET_HATCHED_EGGS_CR:
```
 
the compiler will inline them in any case